routes/api/v1/forecast.js

```diff
@@ -0,0 +1,124 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../../../models').User;
+var Query = require('../../../models').Query;
+var City = require('../../../models').City;
+var Forecast = require('../../../pojos/forecast');
+
+var pry = require('pryjs');
+const fetch = require('node-fetch');
+//eval(pry.it)
+
+router.get("/", function(req, res, next) {
+  let queryLocation = req.query.location;
+  if (req.body.api_key){
```

Comment:
You might be able to add a function to the user model that would handle the API key lookup and be reusable in other places, which would reduce some of the nesting here.

Reply:
Seconded on this